packages/lexical-playground/src/plugins/ContextMenuPlugin/index.tsx

```diff
@@ -183,6 +184,21 @@ export default function ContextMenuPlugin(): JSX.Element {
               .at(-2);
 
             ancestorNodeWithRootAsParent?.remove();
+          } else if ($isNodeSelection(selection)) {
+            const selectedNodes = selection.getNodes();
+            const deletableNodeTypes = new Set([
+              'image',
+              'inline-image',
+              'page-break',
+              'excalidraw',
+              'horizontalrule',
+            ]);
+
+            selectedNodes.forEach((node) => {
+              if (deletableNodeTypes.has(node.getType())) {
+                node.remove();
+              }
+            });
```

Comment:
Any reason not to delete any decorator in the node selection?

```suggestion
            selectedNodes.forEach((node) => {
              if ($isDecoratorNode(node)) {
                node.remove();
              }
            });
```

Reply:
Thank you for the suggestion!